### PR TITLE
[PLAYER-1963] 

### PR DESCRIPTION
**Pull request by AlinaVoronkovaEpam**
Add required video formats

### CODE REVIEW

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ void completePlayerSetup(boolean asked) {
 			skinLayout = (OoyalaSkinLayout) findViewById(R.id.ooyalaSkin);
 			// Create the OoyalaPlayer, with some built-in UI disabled
 			PlayerDomain playerDomain = new PlayerDomain(domain);
-			Options options = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).build();
+			Options options = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).setBypassPCodeMatching(true).build();
```
Comment (brol1dev) [inline OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java]: No need to add bypass by default, just make sure you add the correct pcodes with embed codes. If you don't know how to get the pcode let me know.

**Review by brol1dev — CHANGES_REQUESTED** [review]

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java [hunk 2]
```diff
@@ -38,6 +37,12 @@ public void onCreate(Bundle savedInstanceState) {
     setTitle(getName());
 
     selectionMap = new LinkedHashMap<String, PlayerSelectionOption>();
+    selectionMap.put("HLS VOD", new PlayerSelectionOption("A0NW1vYTE6hCIME00w2On3_iHKOCE6Z7", "x2aDkyOt2q3WtOCp-krSyDffASzL", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
+    selectionMap.put("eHLS VOD", new PlayerSelectionOption("hneHVnMzE6sEsCT9DE1v_yWi0o-9aCTw", "x2aDkyOt2q3WtOCp-krSyDffASzL", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
+    selectionMap.put("DASH VOD", new PlayerSelectionOption("g3aDUzODE6EWDhHR_FDxuCSPRTBCE1Gq", "hjZ2gyOn8Y0z7pvLjDdvaibqJ-dR", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
+    selectionMap.put("DRM DASH", new PlayerSelectionOption("Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm", "FoeG863GnBL4IhhlFC1Q2jqbkH9m", "http://www.ooyala.com", OoyalaSkinPlayerActivity.class));
```
Comment (brol1dev) [inline OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java]: The DRM one is not playing yet. You need to add this one in the ContentProtectionSampleApp anyway.

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java (hunk 1 above)
Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java]: I took, for instance, DRM DASH from [this list ](https://docs.google.com/spreadsheets/d/1EWh5a5UOgLHlZ78OdBGOKuVjwoed14Nms0q_BsrIOvs/edit#gid=1633781289) the first link TC01 [DRM DASH](https://debug.ooyala.com/ea/index.html?ec=Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm&pbid=8287c6992b2d40e6ab182fb2c5014c17&pcode=FoeG863GnBL4IhhlFC1Q2jqbkH9m&core_player=%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fcore.min.js&video_plugins=%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fvideo-plugin%2Fbit_wrapper.min.js&html5_skin=%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fhtml5-skin.min.js&skin_asset=%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fhtml5-skin.min.css&skin_config=%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fskin.json).
**Embed code** Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm
**Pcode** FoeG863GnBL4IhhlFC1Q2jqbkH9m
And than I got the error: "The provided pcode does not match the embed code owner". Is the pcode correct?
I guess I chose the correct one.

Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java]: Also eHLS doesn't work without setBypassPCodeMatching(true).
[eHLS res](http://debug.ooyala.com/ea/index.html?ec=hneHVnMzE6sEsCT9DE1v_yWi0o-9aCTw&pbid=378512f55dd44fc287d0fa609353c268&pcode=x2aDkyOt2q3WtOCp-krSyDffASzL&core_player=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fcore.js&video_plugins=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fvideo-plugin%2Fbit_wrapper.min.js&html5_skin=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fhtml5-skin.min.js&skin_asset=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fhtml5-skin.min.css&skin_config=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fskin.json&options=%7B%22platform%22%3A%22html5%22%7D)

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java (hunk 2 above)
Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java]: Moved the DRM to the ContentProtectionSampleApp. Thanks.

File: ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/lists/ContentProtectionListActivity.java [hunk 3]
```diff
@@ -37,6 +36,7 @@ public void onCreate(Bundle savedInstanceState) {
     //Populate the embed map
 //    selectionMap.put("Adobe Pass Integration", new PlayerSelectionOption("VybW5lODrJ0uM9FBo7XTT6TNjTJfr_7G", "B3MDExOuTldXc1CiXbzAauYN7Iui", "http://www.ooyala.com", AdobePassSampleAppAcitivity.class) );
     selectionMap.put("Ooyala Player Token", new PlayerSelectionOption("0yMjJ2ZDosUnthiqqIM3c8Eb8Ilx5r52", "c0cTkxOqALQviQIGAHWY5hP0q9gU", "http://www.ooyala.com", OoyalaPlayerTokenPlayerActivity.class) );
+    selectionMap.put("DRM DASH", new PlayerSelectionOption("Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm", "FoeG863GnBL4IhhlFC1Q2jqbkH9m", "http://www.ooyala.com", OoyalaPlayerTokenPlayerActivity.class));
```
Comment (brol1dev) [inline ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/lists/ContentProtectionListActivity.java:39]: Are you able to play this asset? I haven't been able to play it.

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java (hunk 2 above)
Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaSkinListActivity.java]: Could you provide info about `APIKEY` and `SECRET` and how can I generate it?

File: ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/lists/ContentProtectionListActivity.java (hunk 3 above)
Comment (AlinaVoronkovaEpam) [reply ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/lists/ContentProtectionListActivity.java:39]: Yes, I'm able to play this DRM DASH. As far as I know not all of Android devices support DRM so you can check, for example, https://www.ghacks.net/2016/12/03/check-your-android-devices-drm-capabilities/

**Comment by brol1dev** [comment]
We verified the assets in this PR and they are all playing for ExoPlayer 2. I'm closing the ticket. 

Thanks @AlinaVoronkovaEpam

**Pull request closed without merging**